linux_sql/sql/ddl.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE IF NOT EXISTS PUBLIC.host_info (
+  id SERIAL NOT NULL,
+  hostname VARCHAR NOT NULL,
+  cpu_number INT,
+  cpu_architecture VARCHAR,
+  cpu_mhz DECIMAL,
+  L2_cache INT,
+  total_mem INT,
+  "timestamp" TIMESTAMP NOT NULL,
+  PRIMARY KEY(id)
+);
+CREATE TABLE IF NOT EXISTS PUBLIC.host_usage (
+  "timestamp" TIMESTAMP NOT NULL,
+  host_id SERIAL NOT NULL,
+  memory_free INT,
+  cpu_idle INT,
+  cpu_kernel INT,
+  disk_io INT,
+  disk_available INT,
+  FOREIGN KEY(host_id) REFERENCES host_info(id)
+);
+INSERT INTO host_info (
```

Comment:
Insert script should be separate from the database definition script (creating the tables etc).

Reply:
will fix this!!